### PR TITLE
Feature distribution

### PR DESCRIPTION
**Pull request by Sidu28**
Added random forest important feature visualizations

### CODE REVIEW

File: random_forest.py [hunk 1]
```diff
@@ -40,7 +42,14 @@ def run(self):
...
 
-        x = self.X.dropna(axis=1)
+        x = self.X
+        fig = plt.figure(figsize=(10, 5))
+        plt.barh(x.columns[14:], important_features[14:])
+        plt.xlabel("Feature Importance According to Random Forest")
+        plt.ylabel("")
+        plt.title("RF Importance Score")
+        plt.show()
+        print(important_features)
```
Comment (mananshah99) [inline random_forest.py:52]: What is imporant_features? Perhaps print a little description of what's going on before printing this?

File: random_forest.py [hunk 2]
```diff
@@ -152,11 +161,11 @@ def optimize_hyperparameters(self, train_X, train_y):
...
 
-    parser = argparse.ArgumentParser(description="Execute feature extraction for an input PE file")
-    parser.add_argument('file', type=str, help="file containing PE file features")
-    args = parser.parse_args()
+    #parser = argparse.ArgumentParser(description="Execute feature extraction for an input PE file")
+    #parser.add_argument('file', type=str, help="file containing PE file features")
+    #args = parser.parse_args()
 
-    RFPredictor(args.file).run()
+    RFPredictor('features_good_bad.csv').run()
```
Comment (mananshah99) [inline random_forest.py:168]: Why has the argument parser been removed?

**Review by mananshah99 — CHANGES_REQUESTED** [review]
Please remove all files in .dea and features/__pycache__/*. Also, why has the argument parser been removed in `random_forest.py`? The rest of the code looks great!

File: random_forest.py (hunk 1 above)
Comment (Sidu28) [reply random_forest.py:52]: That was for debugging, I have removed it now.

File: random_forest.py (hunk 2 above)
Comment (Sidu28) [reply random_forest.py:168]: Again a debugging thing.  Has been restored

**Pull request merged**